src/main/java/com/flightstats/http/HttpTemplate.java

```diff
@@ -364,4 +378,13 @@ public Response delete(URI uri) {
             throw Throwables.propagate(e);
         }
     }
+
+    private HttpEntity buildMultipartEntity(List<Part> parts, Optional<String> separator) {
+        MultipartEntityBuilder entityBuilder = MultipartEntityBuilder.create();
+        entityBuilder.setBoundary(separator.orElse("fava_" + LocalDate.now().format(DateTimeFormatter.ofPattern("YYYY-MM-dd"))));
```

Comment:
I think the default separator should be much more unique.  Like append another 60 random characters or some other entropy onto the end, because the current "fava_2015-11-04" style is short enough to be problematic.